consensus/src/chained_bft/chained_bft_smr_test.rs

```diff
@@ -490,11 +490,12 @@ fn block_retrieval_with_timeout() {
         playground.stop_drop_message_for(&nodes[0].signer.author(), &nodes[2].signer.author());
 
         playground
-            .wait_for_messages(2, NetworkPlayground::proposals_only)
+            .wait_for_messages(2, NetworkPlayground::votes_only)
```

Comment:
because vote today will trigger retrieval too?

Reply:
It's more about the logic of the test. I was reading it because it was failing for an unrelated reason and realized that what we need here is an indication that both nodes has received the proposal (2 vote messages).